trussworks/react-uswds
# fix: a11y: adds css to hide search clear button that causes accessibility issues

# Summary

## Related Issues or PRs

**USWDS issue:** https://github.com/uswds/uswds/issues/5277

**Chromium issue:** https://bugs.chromium.org/p/chromium/issues/detail?id=1147314

## How To Test

`yarn storybook`

http://localhost:9009/?path=/story/components-search-search--default-search 

1. type in the search box in Chrome
2. make sure the little X icon does not show up

### Screenshots (optional)
before:
![image](https://github.com/trussworks/react-uswds/assets/59394696/cfb378de-f6c9-488a-b521-a252f76b13a7)

after:
<img width="364" alt="image" src="https://github.com/trussworks/react-uswds/assets/59394696/334dcce7-8e21-4e53-933f-245ae90405fb">
